scout/quickstart.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Docker Scout quickstart
+keywords: scout, supply chain, vulnerabilities, packages, cves, scan, analysis, analyze
+description: >
+  Learn how to get started with Docker Scout to analyze images and fix vulnerabilities
+redirect_from:
+  - /atomist/get-started/
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout analyzes image contents and generates a detailed report of packages
+and vulnerabilities that it detects. It can provide you with
+suggestions for how to remediate issues discovered by image analysis.
+
+
+This guide takes a vulnerable container image and shows you how to use Docker
+Scout to identify and fix the vulnerabilities, compare image versions over time,
+and share the results with your team.
+
+## Setup
+[This example project](https://github.com/docker/scout-demo-service) contains 
+a vulnerable Node.js application that you can use to follow along.
+
+1. Clone its repository:
+
+    ```shell
+    git clone https://github.com/docker/scout-demo-service.git
+    ```
+
+2. Move into the directory:
+
+    ```shell
+    cd scout-demo-service
+    ```
+
+3. Build the image, naming it to match the organization you will push it to, 
+and tag it as "v1":
+
+    ```shell
+    docker build -t <org-name>/scout-demo:v1 .
+    ```
+
+4. Create and push the repository on Docker Hub:
+
+    ```shell
+    docker push <org-name>/scout-demo:v1
+    ```
+
+    > **Important**
+    >
+    > Make sure you login to the Docker CLI or Docker Desktop before pushing.
+    > {: .important}
+
+## Enable Docker Scout
+Docker Scout analyzes all local images by default. To analyze images in
+remote repositories, you need to enable it first.
+[You can do this from Docker Hub, the Docker Scout Dashboard, and CLI](/scout).
+
+1. Use the Docker CLI [`scout repo enable`](/engine/reference/commandline/scout_repo_enable)
+command to enable analysis on an existing repository with the following command:
+
+    ```shell
+    docker scout repo enable <org-name>/scout-demo
+    ```
+
+## Analyze image vulnerabilities
+
+After building, you can use Docker Desktop or the `docker scout` CLI command 
+to see vulnerabilities detected by Docker Scout.
+
+1. Using Docker Desktop, select the image name in the **images** tab to see 
+the image layer view. In the image hierarchy section, you can see which layers 
+introduce vulnerabilities and the details of those.
+
+2. Select layer 5 to focus on the vulnerability introduced in that layer.
+
+3. Toggle the disclosure triangle next to **express 4.17.1** and then the CVE ID (in this case, "CVE-2022-24999⁠") to see details of the
+vulnerability.
+
+4. You can also use the Docker CLI to see the same results.
```

Comment:
This is another way to see the details of the vulnerability. Should this be a new step?

Reply:
@usha-mandya I went back and forth on this myself, happy to make it part of the same